### ISSUE TITLE
Modify 'mepo commit' to look more like 'git commit'

### ISSUE THREAD

**Issue by pchakraborty** [GEOS-ESM/mepo#17]
Currently `mepo commit` usage is as follows:
```
mepo commit message comp-name [comp-name ...]
```
It would be better if `mepo commit` resembled `git commit` more -
```
mepo commit [-m message] comp-name [comp-name]
```
That is, the commit message is specified via `-m` flag. In the absence of the flag, $EDITOR is opened to capture the commit message.

**Comment by pchakraborty** [comment]
Here is an example of how $EDITOR can be opened as a subprocess: https://stackoverflow.com/a/3076863

**Comment by mathomp4** [comment]
Fixed by #49
